src/Element/SendSats.tsx

```diff
@@ -77,6 +77,8 @@ export default function LNURLTip(props: LNURLTipProps) {
   const { formatMessage } = useIntl();
   const publisher = useEventPublisher();
   const horizontalScroll = useHorizontalScroll();
+  const canComment =
+    (payService?.commentAllowed ?? 0) > 0 || payService?.nostrPubkey;
```

Comment:
Having a `nostrPubkey` doesnt mean you can pass comment, only `commentAllowed > 0`

Reply:
zaps do allow comments tho, they are part of the zap not the invoice comment afaict

Reply:
Yes but this is for the `?comment=` param, not the comment of the zap note, i guess it doesn't matter if you pass comment arg and its not supported for zaps endpoint

Reply:
will not send param if not supported by lnurl endpoint, good point